safe/gui/tools/test/test_options_dialog.py

```diff
@@ -0,0 +1,30 @@
+# coding=utf-8
+"""Test InaSAFE Options Dialog"""
+
+import unittest
+import logging
+
+# This import is to enable SIP API V2
+# noinspection PyUnresolvedReferences
+import qgis  # pylint: disable=unused-import
+
+from safe.gui.tools.options_dialog import OptionsDialog
+from safe.test.utilities import get_qgis_app
+
+QGIS_APP, CANVAS, IFACE, PARENT = get_qgis_app()
+LOGGER = logging.getLogger('InaSAFE')
+
+
+class TestOptionsDialog(unittest.TestCase):
+    """Test Options Dialog"""
+
+    def test_setup_dialog(self):
+        """Test Setup Options Dialog."""
+        dialog = OptionsDialog(PARENT, IFACE)
+        pass
```

Comment:
No `assert`, is-it normal ? Can we remove the test ? If this is a test, please explain it with a comment in the code.